### ISSUE TITLE
Placing down rail in blueprint mode breaks after a short delay in survival mode

### ISSUE THREAD

**Issue by RRyyas** [TeamOpenIndustry/ImmersiveRailroading#798]
**Describe the bug**  
The rails breaks itself about a few seconds after it has been placed down from blueprint mode. This does not happen in creative mode, and there's no obstacles.

**How to reproduce**  

 1. Get the required materials for placing down a 10 block straight standard-gauge track in survival mode
 2. Go on survival mode
 3. Place a track blueprint of a 10 block straight standard-gauge track
 4. Shift-left click the blueprint

Reproduction steps are also in the gfycat link.
 
**Expected behavior**  
 The rail places down from a blueprint without breaking for no reason

**Screenshots**  
 https://gfycat.com/reliablewhirlwindherring

**System Information**  
 OS: Windows 10
 <br>System Memory: 32 GB
 <br>Allocated Memory: 16500 MB
 <br>CPU: Intel i7-6700k
 <br>GPU: GTX 1080 Ti
 <br>IR-Version: 1.7.3_1.12.2
 <br>Forge-Version: 1.12.2 - 14.23.5.2847

**Modlist**  
 ImmersiveEngineering-0.12-88
 TrackAPI-1.2 etc.
_EDIT: This issue is caused by the Multi Mine mod_

**Additional context**  
I have changed the "trackFloatingPercent" option to 33.3% as a debugging measure, with no difference.

**Comment by andrboot** [comment]
Please include your entire mod-list.

**Comment by RRyyas** [comment]
Because it's a large modpack, i can't list them all here, but however i narrowed the issue down to one single mod: MultiMine-1.12.2. That mod causes the rails to break after a short delay.

**Comment by Melikoth** [comment]
I had this happen a few times recently in a custom pack (FizzPack). Playing survival mode.

It occurred for me with the switch track when making a 22.5 degree turn in fixed mode with a length of 30, locked right. It would sit there fine, then after some time break. This happened a few times, but the timing on each break was not consistent. Cleared out everything around it to make sure nothing was interfering, but it made no difference.

I decided to try a 45 degree angle to see if it would stop breaking and it broke almost immediately every time! After 3-4 attempts that broke immediately, I switched back to 22.5 and laid it back down. Oddly, this time it didn't break and as of now has held for at least 5 days.

In the meantime, I had another occurrence where a 22.5 track switch kept breaking. I added a few blocks to the outer edge of the curve on the switch (since the fill layer remains when the rails break) and placed it down again. This time it did not break.

It seems like when deciding to lay the track, it misses some spots for fill layers that it later decides are important when making a check to determine if the track should break.
